backend/src/core/config.ts

```diff
@@ -15,6 +15,137 @@ const rdsCa = fs.readFileSync(path.join(__dirname, '../assets/db-ca.pem'))
  *    format: 'required-string',
  *    sensitive: true,
  */
+interface ConfigSchema {
+  env: string
+  APP_NAME: string
+  aws: {
+    awsRegion: string
+    awsEndpoint: null
+    logGroupName: string
+    uploadBucket: string
+    secretManagerSalt: string
+  }
+  database: {
+    databaseUri: string
+    databaseReadReplicaUri: string
+    dialectOptions: {
+      ssl: {
+        require: boolean
+        rejectUnauthorized: {
+          valueOf: any
```

Comment:
is this a problem? I just copied from the type inference but I hate `any` @stanleynguyen

Reply:
Oh iinw `rejectUnauthorized` should just be a boolean flag (which we defaulted to `true`)

Reply:
I tried that but somehow TypeScript doesn't agree hmm

nvm just merge first